FirebaseMLModelDownloader/Sources/ModelDownloadConditions.swift

```diff
@@ -14,10 +14,10 @@
 
 import Foundation
 
-/// Model download conditions.
+/// Conditions that need to be met to start a model file download.
 public struct ModelDownloadConditions {
   /// Allow model downloading on a cellular connection. Default is `YES`.
   public var allowsCellularAccess = true
-  /// Public init.
+
   public init() {}
```

Comment:
Just looking at this, I'm wondering if a more clear API is to have it as `let allowsCellularAccess`, have the initializer take in an `allowsCellularAccess` parameter with the default value of `true`. WDYT? Then the `Default is 'true'` is in the init code.

Reply:
Ah that makes more sense. I'll update this here and the API doc.